src/binfhe/lib/rgsw-acc-lmkcdey.cpp

```diff
@@ -167,7 +167,6 @@ RingGSWEvalKey RingGSWAccumulatorLMKCDEY::KeyGenLMKCDEY(const std::shared_ptr<Ri
 
     DiscreteUniformGeneratorImpl<NativeVector> dug;
     NativeInteger Q{params->GetQ()};
-    dug.SetModulus(Q);
```

Comment:
would like to make sure that dug doesn't need to be constructed with the modulus "Q".